examples/js/loaders/ColladaLoader2.js

```diff
@@ -2452,7 +2452,9 @@ THREE.ColladaLoader.prototype = {
 					case 'axis':
 						var param = child.getElementsByTagName( 'param' )[ 0 ];
 						data.axis = param.textContent;
-						data.jointIndex = parseInt( data.axis.split( 'joint' ).pop().split( '.' )[ 0 ] );
+						var tmpJointIndex = data.axis.split( 'inst_' ).pop().split( 'axis' )[ 0 ];
+						tmpJointIndex = tmpJointIndex.substr(0, tmpJointIndex.length - 1);
```

Comment:
Just a code style issue:
```javascript
tmpJointIndex.substr( 0, tmpJointIndex.length - 1 );
```

Reply:
BTW: Can you directly assign this statement to `data.jointIndex`? Then you would save the next line...

Reply:
Sure can! :+1: